Sources/Profile-View.php

```diff
@@ -384,7 +384,7 @@ function fetch_alerts($memID, $all = false, $counter = 0, $pagination = array(),
 		}
 	}
 
-	$user_info = !empty($user_old) ? $user_old : null;
+	$user_info = !empty($user_old) ? $user_old : $user_info;
```

Comment:
Why you set this to user_info when old is null?

Reply:
The old `$user_info` isn't null. `$user_old` isn't used in this case, and so isn't set. Therefore, all this line used to do was set a populated `$user_info` to null, for no useful reason.

Reply:
Sure it's usefull it's resetet the state of before 222,
you should fix the issue and not this logic here.

Reply:
The logic error was the issue though.

Reply:
Why is $user_old empty?
It's very dangarous you implemenation smf is not designed to work this way.

Reply:
$user_info should not be empty. That line I corrected was making it null and causing issues, so I fixed it to not be null.

Reply:
Please read the pr message of this line of code: https://github.com/SimpleMachines/SMF2.1/pull/4754
The issue is that $user_old is empty,
that needs to be fixed.